### ISSUE TITLE
AVT 2 - Time Picker time placeholder fails color contrast 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4254]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React

## Detailed Description
Tab to the dropdown list 
Press Enter to open the listbox
**Expected Result:** Listbox should display using the Enter key. 
See [ARIA Authoring Practices for s collapsible listbox](https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html).

Using Color Contrast Analyzer compare the placeholder text (hh:mm) for the time against the light gray background and it fails. See screenshot below:
<img width="1429" alt="Screen Shot 2019-10-08 at 9 13 57 AM" src="https://user-images.githubusercontent.com/21676914/66403658-91b95600-e9ac-11e9-8292-96a8162a4eea.png">
.

**Comment by dakahn** [comment]
@snidersd Hey, so I see the APG example referenced, but if I check the suggested practices I don't see `enter` listed even as a recommendation. Seems like the example and the spec have deviated here?
https://www.w3.org/TR/wai-aria-practices-1.1/#listbox_kbd_interaction

Further, I would expect enter to maybe throw open the list if we had a button as the main control, but from a markup standpoint this is just a good ol' `<select>` a la 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select

**Comment by snidersd** [comment]
@dakahn I agree. There are definitely deviations. I was looking at the component as a collapsible dropdown and not a simple select.

**Comment by abbeyhrt** [comment]
@snidersd would you be okay with saying this issue could be closed if the placeholder color contrast is addressed?

**Comment by snidersd** [comment]
@abbeyhrt Yep, that works for me. Thx
